src/hooks/usePageTitlePriceUpdates.ts

```diff
@@ -17,13 +21,17 @@ export const usePageTitlePriceUpdates = () => {
   const { isNotTablet } = useBreakpoints();
   const id = useAppSelector(getCurrentMarketId);
   const oraclePrice = useAppSelector(getCurrentMarketOraclePrice);
+  const { tickSizeDecimals } = useAppSelector(getCurrentMarketConfig, shallowEqual) ?? {};
+
   const orderbookMidMarketPrice = useAppSelector(getCurrentMarketMidMarketPrice);
 
   const price = orderbookMidMarketPrice ?? oraclePrice;
 
   useEffect(() => {
     if (id && price && isNotTablet) {
-      const priceString = price.toLocaleString(selectedLocale);
+      const priceString = price.toLocaleString(selectedLocale, {
```

Comment:
Looks like everywhere else in the app we use bignumber.toFormat to format numbers. I think ideally we would make a helper function that takes a number and an OutputType and returns a string then use that function in both Output and all these places we need strings. 

I'm happy to merge this as is though and I can make a follow up pr.